server/test/support/CfTestHelpers.java

```diff
@@ -119,4 +122,70 @@ public static Http.RequestBuilder requestBuilderWithSettings(
 
     return requestBuilder.attr(CIVIFORM_SETTINGS_ATTRIBUTE_KEY, settingsMap.build());
   }
+
+  public static class ResultWithFinalRequestUri {
```

Comment:
**nitpick (blocking):** please add javadoc for this

great solution!

Reply:
Done